pom.xml

```diff
@@ -17,6 +17,14 @@
 
    <properties>
       <jmockit.version>1.49</jmockit.version>
+      <sonar.organization>usdot-jpo-ode-1</sonar.organization>
+      <sonar.host.url>https://sonarcloud.io</sonar.host.url>
+      <!-- JaCoCo Properties -->
+      <jacoco.version>0.8.8</jacoco.version>
+      <sonar.java.coveragePlugin>jacoco</sonar.java.coveragePlugin>
+      <sonar.dynamicAnalysis>reuseReports</sonar.dynamicAnalysis>
+      <sonar.coverage.jacoco.xmlReportPaths>${project.basedir}/target/site/jacoco/jacoco.xml</sonar.coverage.jacoco.xmlReportPaths>
+      <sonar.language>java</sonar.language>
```

Comment:
Is this being used? It's duplicating stuff in the sonar properties file like organization